_health-care/_personal_information/_name_and_general_information.html

```diff
@@ -48,11 +48,20 @@
 
     <div class="row">
       <div class="small-3 columns">
+        <label for="veteran_date_of_birth">Date of Birth</label>
+      </div>
+      <div class="small-9 columns">
+        <input type="text" name="veteran[date_of_birth]" />
```

Comment:
This probably isn't the best input type for a date field, as you'll probably get all kinds of weird formats and submissions (unless you want to include some sort of client-side validation before anything is submitted). HTML5 has a `input type="date"`, but that's not supported in IE yet. Rails has a form helper called `date_select`, which creates 3 individual dropdowns (month, day, year), and you could use that with form_for. But we could probably wait to implement that until later.